configuration/display_rotation.md

```diff
@@ -9,10 +9,18 @@ If you are running the Raspberry Pi desktop then rotation is achieved by using t
 It is also possible to change these settings using the command line `xrandr` option. The following commands give 0°, -90°, +90° and 180° rotations respectively. 
 
 ```bash
-DISPLAY=:0 xrandr --output HDMI1 --rotate normal
-DISPLAY=:0 xrandr --output HDMI1 --rotate left 
-DISPLAY=:0 xrandr --output HDMI1 --rotate inverted
-DISPLAY=:0 xrandr --output HDMI1 --rotate right
+xrandr --output HDMI-1 --rotate normal
+xrandr --output HDMI-1 --rotate left 
+xrandr --output HDMI-1 --rotate right
+xrandr --output HDMI-1 --rotate inverted
+```
+
+Note that the `--output` entry specifies to which device the rotation applies. You can determine the device name by simply typing `xrandr` on the command line which will display information, including the name, for all attached devices.
```

Comment:
Thanks for this clarification! Up until now I'd been assuming that "HDMI-1" was referring to the secondary HDMI1 micro-HDMI port on the Pi4B, but by typing `xrandr` I now see that it's actually referring to the "primary" display i.e. the HDMI0 micro-HDMI port.